README.md

```diff
@@ -1,6 +1,7 @@
 # Ethereum Execution Client Specifications
 
 [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/ethereum/execution-specs/badge)](https://www.gitpoap.io/gh/ethereum/execution-specs)
+[![codecov](https://codecov.io/gh/ethereum/execution-specs/graph/badge.svg?token=${{ secrets.CODECOV_TOKEN }})](https://codecov.io/gh/ethereum/execution-specs)
```

Comment:
I don't think GitHub will substitute a secret here. Are you sure this is the token we're supposed to use?